### DISCUSSION TITLE
Issues with nested components on lazy parents

### DISCUSSION THREAD

**Discussion by OccTherapist** [livewire/livewire#7552] [1. Help] [answered]
Has anyone of you noticed issues on Livewire components with the lazy attributes?

I am making the experience that wire:model on nested components stops working when I add lazy, to the parent.

Why is this?

**Comment by joshhanley** [comment]
@OccTherapist can you share some code that demonstrates what you mean?

**Reply by OccTherapist** [reply, marked as answer]
I figured it out myself by now. I found a GitHub discussion where someone mentioned I have to wrap components in an empty div to make pagination and other wires work.

**Reply by joshhanley** [reply]
@OccTherapist ah yep, sweet! Yeah always need to ensure there is one root element for a Livewire component.
